lib/jdomodels/src/main/java/org/sagebionetworks/repo/model/dbo/dao/DBOFileHandleDaoImpl.java

```diff
@@ -287,9 +292,9 @@ public FileHandleResults getAllFileHandles(Iterable<String> ids, boolean include
 	}
 
 	@Override
-	public long getS3objectReferenceCount(String bucketName, String key) {
+	public long getNumberOfReferencesToFile(String metadataType, String bucketName, String key) {
```

Comment:
This method counts the number of filehandles that point to the same underlying file. The changes allow the method to work with both S3 and Google Cloud file handles.